samcli/commands/build/build_context.py

```diff
@@ -155,7 +155,7 @@ def __exit__(self, *args):
     def get_resources_to_build(self):
         return self.resources_to_build
 
-    def run(self):
+    def run(self, sync=False):
```

Comment:
Can we have a separate PR for this?

Reply:
The integration tests are dependent on this PR's integration test changes but sure I can have two PR's one with just he integration test changes and the other integration test + ux, after integration test merge I can do a merge and have the ux changes merged as a separate PR